chainbase/src/main/java/org/tron/core/db2/core/SnapshotManager.java

```diff
@@ -233,6 +233,12 @@ public synchronized void commit() {
     }
 
     --activeSession;
+
+    dbs.forEach(db -> {
```

Comment:
Why set the reload logic in `commit()`, `commit()` only invoked by the end of pushing a block, not a transaction.

Are there other reasons for doing this, can you explain?

Reply:
+1

Reply:
This is because if the transaction ends without a commit,
Revoke is needed at this time,
session.close is called,
Then call snapshotManager.revoke,
Then there is no need to perform reload logic.
So the reload logic is only executed when the transaction commits. 

@tomatoishealthy @halibobo1205